### DISCUSSION TITLE
Github returns wrong fingerprint

### DISCUSSION THREAD

**Discussion by jorick23** [community/community#80960] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I started having problems with my repository ("Permission denied (publickey)", so I generated a new key (multiple times) and added it to Github.  But the key doesn't work because the fingerprint for my new key doesn't match the fingerprint supplied by Github.  Is there anyway to fix this?

My key's fingerprint: SHA256:UjkS...8zwg
Github returns: SHA256:+DiY...COqU  (always, regardless of how many times I generate a new key)

I've followed Github's instructions on how to make a new key.  I delete the old key from Github before I add the new one.

**Comment by ananuness** [comment, marked as answer]
Hii! You must be having the problem described on this [page](https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-denied-publickey). You can follow this step by step to check what you might be doing wrong. If so, you can respond here and I can help. Also, the key you're seeing is one of those [Github's SSH keys](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/githubs-ssh-key-fingerprints).

**Reply by jorick23** [reply]
Nothing worked until I went to Github's SSH Keys and added them to my known_hosts file.  This was the solution!  But I still wonder why it stopped working since I never did anything to the .ssh directory.

**Reply by ananuness** [reply]
It happened to me too! I looked more about why this could happen and I only found this link I sent talking about permission denied and someone saying that it could be due to some forgotten step when generating the ssh key the first time resulting in the know_hosts file not being filled out correctly. But I'm glad everything went well!
